packages/analytics-client-common/src/attribution/campaign-tracker.ts

```diff
@@ -42,7 +42,10 @@ export class CampaignTracker implements ICampaignTracker {
     this.initialEmptyValue = options.initialEmptyValue ?? EMPTY_VALUE;
   }
 
-  isNewCampaign(current: Campaign, previous: Campaign) {
+  isNewCampaign(current: Campaign, previous: Campaign | undefined) {
+    if (!previous) {
```

Comment:
we need to consider `isReferrerExcluded` also

Reply:
for example, if they add google.com to excluded referrer and user comes from google.com, then even with no previous campaign, we will not track

Reply:
Oh, I see. Will change